Source/displayinfo/DisplayInfo.cpp

```diff
@@ -178,7 +207,7 @@ POP_WARNING()
 
     uint32_t RegisterOperationalStateChangedCallback(displayinfo_operational_state_change_cb callback, void* userdata)
     {
-        uint32_t result = Core::ERROR_ALREADY_CONNECTED;
+        uint32_t result = displayinfo_status::DISPLAYINFO_ERROR_ALREADY_CONNECTED;
```

Comment:
the type is now the enum not uint32_t :)

Reply:
done